src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx

```diff
@@ -843,7 +547,7 @@ export class Flyout extends Component<FlyoutProps, FlyoutState> {
           <EuiButtonEmpty
             onClick={close}
             size="s"
-            disabled={status === 'loading' || (isLegacyFile === false && status === 'success')}
+            disabled={status === 'loading' || status === 'success'}
```

Comment:
`status` is typed as a string but in the code we assign it as one of `idle`, `loading`, `error` or `success`. Maybe as a follow up we can improve the type to only be one of those four?